components/automate-ui/src/app/entities/infra-nodes/infra-nodes.requests.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { environment as env } from 'environments/environment';
+import { NodesSuccessPayload } from './infra-nodes.actions';
+import { InterceptorSkipHeader } from 'app/services/http/http-client-auth.interceptor';
+
+@Injectable()
+export class InfraNodeRequests {
+
+  constructor(private http: HttpClient) { }
+
+  public getNodes(server_id: string, org_id: string):
+  Observable<NodesSuccessPayload> {
```

Comment:
Let's only break a line if necessary.
```suggestion
  public getNodes(server_id: string, org_id: string): Observable<NodesSuccessPayload> {
```